src/dbus/server/CMakeLists.txt

```
@@ -26,14 +26,26 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
+add_custom_target(otbr-dbus-instropect
+  COMMAND
+    ${CMAKE_CURRENT_SOURCE_DIR}/generate_introspect.sh 
+)
```

use configure time generation instead of compile time.
```suggestion
file(READ introspect.xml OTBR_DBUS_INTROSPECT)
file(WRITE introspect.hpp "R\"(")
file(APPEND introspect.hpp OTBR_DBUS_INTROSPECT)
file(APPEND introspect.hpp ")\"")
```